src/cljs/athens/events.cljs

```diff
@@ -284,6 +284,22 @@
        :dispatch [:editing/uid child-uid]})))
 
 
+(reg-event-fx
+  :page/make-shortcut
+  (fn [_ [_ uid]]
+    (let [sidebar-ents (d/q '[:find ?e
+                              :where
+                              [?e :page/sidebar _]]
+                            @db/dsdb)]
+      {:transact! [{:block/uid uid :page/sidebar (count sidebar-ents)}]})))
+
+
+(reg-event-fx
+  :page/unmake-shortcut
+  (fn [_ [_ uid]]
+    {:transact! [[:db/retract [:block/uid uid] :page/sidebar]]}))
```

Comment:
Honestly really impressed you used `:block/uid` as the identifier for these two queries. I didn't know that was possible / hand't thought to do that.

Reply:
Thoughts on `:page/add-shortcut` and `:page/remove-shortcut`? That's the label of the buttons